webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var webpack = require('webpack');
+var fs = require('fs');
+var copyright = fs.readFileSync('lib/copyright.frag', 'utf8');
+
+module.exports = {
+    entry: './lib/index.js',
+    output: {
+        path: path.join(__dirname, 'dist', 'browser'),
```

Comment:
https://github.com/webpack/docs/wiki/configuration#outputpath

`The output directory as absolute path (required).`